nemo/collections/asr/parts/utils/manifest_utils.py

```diff
@@ -33,6 +33,79 @@
 from nemo.utils.data_utils import DataStoreObject
 
 
+def get_ctm_line(
+    source: str,
+    channel: int,
+    beg_time: float,
+    duration: float,
+    token: str,
+    conf: float,
+    type_of_token: str,
+    speaker: str,
+    NA_token: str = 'NA',
+    UNK: str = 'unknown',
+    default_channel: str = '1',
+    output_precision: int = 2,
+) -> str:
+    """
+    Get a line in  Conversation Time Mark (CTM) format. Following CTM format appeared in `Rich Transcription Meeting Eval Plan: RT09` document.
+    
+    CTM Format: 
+        <SOURCE><SP><CHANNEL><SP><BEG-TIME><SP><DURATION><SP><TOKEN><SP><CONF><SP><TYPE><SP><SPEAKER><NEWLINE>
+    
+    Reference: 
+        https://web.archive.org/web/20170119114252/http://www.itl.nist.gov/iad/mig/tests/rt/2009/docs/rt09-meeting-eval-plan-v2.pdf
+
+    Args:
+        source (str): <SOURCE> is name of the source file, session name or utterance ID
+        channel (int): <CHANNEL> is channel number defaults to 1
+        beg_time (float): <BEG_TIME> is begin time of the word
```

Comment:
I kind of feel that `start_time` is easier to understand than `beg_time`, without needing to look at the documentations

Reply:
I agree it is easier to understand and probably `beg_time` was an ill-suited choice in the initial .ctm specification. 
On the other hand, if someone is using this function without looking at the specification maybe he should not use it. 
IDK I think if we use the original names it is less confusion (even if yeah, it is less intuitive).

Reply:
Changed `beg_time` to `start_time` and then mentioned the difference in the docstring.